### PR TITLE
API: expose domains.account

### PR DESCRIPTION
**Pull request by zeha**
As requested by @jpmens in #2188

### CODE REVIEW

File: pdns/backends/gsql/gsqlbackend.cc [hunk 1]
```diff
@@ -270,10 +272,25 @@ bool GSQLBackend::setKind(const string &domain, const DomainInfo::DomainKind kin
...
+{
+  try {
+    d_UpdateAccountOfZoneQuery_stmt->
+            bind("account", account)->
+            bind("domain", toLower(domain))->
+            execute()->
+            reset();
+  }
+  catch (SSqlException &e) {
+    throw PDNSException("GSQLBackend unable to set kind of domain \""+domain+"\": "+e.txtReason());
```
Comment (mind04) [inline pdns/backends/gsql/gsqlbackend.cc]: Please replace kind by account in this error

Comment (zeha) [inline pdns/backends/gsql/gsqlbackend.cc]: Fixed, ta.

**Comment by cmouse** [comment]
goracle not supported? =)

**Comment by jpmens** [comment]
Thank you. I will have to wait for merge + autotest packages for testing.

**Comment by jpmens** [comment]
Thanks to @zeha for adding this, thanks to @ahupowerdns for merging, thanks to @Habbie for not bawling out @ahupowerdns for merging ;-)

it works. I tested both GET and POST requests.

**Comment by jpmens** [comment]
@zeha `account` is exposed during GET of a particular zone (`zone/example.net`), but not during GET of `zones` (as suggested in #2203 ). Was this done on purpose?

**Comment by stbuehler** [comment]
I hope not - the code duplication between `apiServerZones` and `fillZone` should be removed...

**Comment by zeha** [comment]
No, I merely forgot that these are two separate code paths.

**Pull request merged**